kms/tpmkms/tpmkms.go

```diff
@@ -468,13 +573,23 @@ func (k *TPMKMS) LoadCertificate(req *apiv1.LoadCertificateRequest) (*x509.Certi
 	return chain[0], nil
 }
 
-// LoadCertificateCertificate loads the certificate chain for the key identified by
+// LoadCertificateChain loads the certificate chain for the key identified by
 // name from the TPMKMS.
 func (k *TPMKMS) LoadCertificateChain(req *apiv1.LoadCertificateChainRequest) ([]*x509.Certificate, error) {
 	if req.Name == "" {
 		return nil, errors.New("loadCertificateChainRequest 'name' cannot be empty")
 	}
 
+	if k.usesWindowsCertificateStore() {
+		chain, err := k.loadCertificateChainFromWindowsCertificateStore(&apiv1.LoadCertificateRequest{
+			Name: req.Name,
+		})
+		if err != nil {
+			return nil, fmt.Errorf("failed loading certificate chain using Windows platform cryptography provider: %w", err)
+		}
+		return chain, nil
```

Comment:
That would be nice and useful, and I've thought about doing that, but decided to skip it (for now). My development flow for Windows is a bit tedious at the moment, and without a proper test environment to test the interactions with a native Windows store, it's hard to guarantee we don't break things. 

We probably want to build it using Wincrypt's [CertGetCertificateChain](https://learn.microsoft.com/en-us/windows/win32/api/wincrypt/nf-wincrypt-certgetcertificatechain) (instead of moving the logic from here to `capi`), which seems to have quite a few parameters with corresponding behavior, so it might take a while to get that part right. The current implementation is intended to return a chain we manage, no matter if it has expired, or has to be retrieved using AIA (which is one of the things `CertGetCertificateChain` supports), so IMO it fits the current needs.